src/frontend/src/lib/components/core/Loader.svelte

```diff
@@ -84,6 +85,11 @@
 			...(AIRDROP ? [initAirdrop()] : [])
 		]);
 
+		const { success: fiatBalanceSuccess } = await loadFiatBalance();
```

Comment:
This is not the correct "place" to load these information. I would have to have a deeper look but, this is not the right place for sure.